File: Math2/Math2.py
import math

x=-1
dx=0.000001
iters=int((1-(-1))/dx)


A=0.
for i in range(iters):
        A=A+math.sqrt(1-x**2)*dx
        x=x+dx
tpi=2*A

print ("pi is approximately %.16f" % (tpi))

File: Math2(C)/math2.c
#include <math.h>
#include <stdio.h>

const int r = 2;
const float dx = 0.001;
const int iters = ((r*2)/(dx));

float A = 0;
int x = -2;

int main() {
	for (int i = 0; i < iters; i++) {
		A = A + sqrt((r * r) - (x *x)) * dx;
		x = x + dx;
	}

	float S = r * r * 2 * A;

	printf("%f", S);

}